packages/paginated-table/Add_Selected_Files_Orange_button_README.md

```diff
@@ -0,0 +1,121 @@
+
+### 1 Use latest or bento-core/paginated-table package version publish by ICDC
+Alternative configuration for ORANGE button
+```
+// 1.0.1-icdc.3
+npm i @bento-core/paginated-table@1.0.1-icdc.3
```

Comment:
@rana22 do we need mention version of paginated table at the time of install? it will make issues for future if we mention version over here in readme.

Reply:
Copied form ICDC branch. Its removed.